src/randoop/test/ValidityCheckingVisitor.java

```diff
@@ -3,30 +3,44 @@
 import randoop.ExceptionalExecution;
 import randoop.ExecutionOutcome;
 import randoop.sequence.ExecutableSequence;
+import randoop.sequence.SequenceExceptionError;
 import randoop.test.predicate.ExceptionPredicate;
 
 /**
  * A {@code ValidityCheckingVisitor} checks for occurrences of exceptions that
  * have been tagged as invalid behaviors as represented by a 
  * {@code ExceptionPredicate}.
+ * Also, either ignores or reports flaky test sequences --- an input 
+ * sequence that throws an exception in a longer test sequence, despite having
+ * run normally by itself.
+ * Ignored flaky sequences are classified as invalid.
+ * Flaky occurrences of {@code OutOfMemoryError} are always treated as invalid.
  */
 public class ValidityCheckingVisitor implements TestCheckGenerator {
 
   private ExceptionPredicate isInvalid;
+  private boolean reportFlaky;
```

Comment:
Please document the variable (what you wrote in the method constructor documentation is fine to appear here too).
"reportFlaky" doesn't really convey that Randoop halts if a flaky test is encountered; maybe this variable should be renamed.